pywebostv/controls.py

```diff
@@ -1,4 +1,10 @@
-from collections import Callable
+try:
+    # begin try for python <= 3.5
+    from collections import Callable
+except ImportError:
+    # after try for python >= 3.10
+    from typing import Callable
```

Comment:
All of this can be simplified to just:

```
from collections.abc import Callable
```

which works in Python >= 3.3.

Sorry for the late comment, just noticed this now.